scripts/test/integration-setup.sh

```diff
@@ -31,8 +31,6 @@ else
 fi
 
 if [ "$DBT_VERSION" = "1.10" ]; then
```

Comment:
We need to handle "$DBT_VERSION" != "1.10" as well..! Otherwise the tests will fail, because those adaptors won't be available.